README.md

```diff
@@ -188,6 +188,11 @@ To refer how to format your .json files to upload as new cases, refer to the exa
 
 ![Uploading New Case](./docs/figures/uploading-new-case.png "Uploading New Case")
 
+### Removing Analyses
```

Comment:
This documentation is necessary to publicize on the primary README since it isn't something we want to encourage folks to do.  You can remove this out.

Reply:
The reference has been removed from the primary README